app/src/org/commcare/views/EntityView.java

```diff
@@ -55,71 +60,107 @@
 
     private boolean mFuzzySearchEnabled = true;
     private boolean mIsAsynchronous = false;
+    private String extraData = null;
 
     /**
      * Creates row entry for entity
      */
     private EntityView(Context context, Detail d, Entity e,
-                       String[] searchTerms, long rowId, boolean mFuzzySearchEnabled) {
+                       String[] searchTerms, long rowId,
+                       boolean mFuzzySearchEnabled, String extraData) {
         super(context);
 
         //this is bad :(
         mIsAsynchronous = e instanceof AsyncEntity;
         this.searchTerms = searchTerms;
         this.renderedGraphsCache = new Hashtable<>();
         this.rowId = rowId;
-        this.views = new View[e.getNumFields()];
-        this.forms = d.getTemplateForms();
-        this.mHints = d.getTemplateSizeHints();
+        this.views = new ArrayList<>(e.getNumFields());

```

Comment:
To be able more easily add/remove views, which is needed to add/remove the callout response data column, for showing the fingerprint match confidence